about/reports/submission_category_by_year.md

```diff
@@ -0,0 +1,18 @@
+# Submissions by Category since 2009+
+
+_See more [arXiv in Numbers](/about/reports/2020_usage)_
```

Comment:
```suggestion
_See more [arXiv in Numbers 2020](/about/reports/2020_usage) and [Usage stats](/stats/main)_
```

Reply:
@afromme Does this look OK to specify 2020 info and link to the main stats page?